### ISSUE TITLE
hey can you explain how to add ads 

### ISSUE THREAD

**Issue by tweakdoor** [emulatorjs/emulatorjs#68]
hey i have added this  EJS_AdUrl = 'MYWEBSITE LINK '; //path to AD page

but ads not showing can you guide please

**Comment by allancoding** [comment]
What version do you have of emulatorjs?

**Comment by tweakdoor** [comment]
the least one i guess and i tryed to add the code inside the game page but not showing

**Comment by allancoding** [comment]
What is the adurl to like what service?

**Comment by tweakdoor** [comment]
to adsence if this what you mean i add my website after  EJS_AdUrl = and nothing shows

**Comment by allancoding** [comment]
what are you testing it on? can I have a link?

**Comment by allancoding** [comment]
The link needs to be to a html page.

**Comment by tweakdoor** [comment]
https://tweakdoor.com/game/test/me.html 

this is my game i added ejs_addurl there and nothing happend

**Comment by allancoding** [comment]
![image](https://user-images.githubusercontent.com/74841470/147250337-2a713b15-538d-4e61-ba64-f9336339d832.png)
It work on my end try a Crl+R to reload the page.

**Comment by tweakdoor** [comment]
<link rel="icon" type="image/png" href="emulator/img/icon.png" />
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
<link rel="stylesheet" href="css/main.css">
<script src="js/main.js"></script>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
<div id="preloader"></div>
<style>
.containerr { 
  top: 0;
  left: 0;
  position: fixed;
  z-index: 99999999;
  opacity: 0;
  transition: opacity .1s ease-in-out;
  -moz-transition: opacity .1s ease-in-out;
  -webkit-transition: opacity .1s ease-in-out;
}

.vertical--center {
  position: fixed;
  top: 0;
  left: 0;
  width: 150px;
  height: 180px;
}
.a{
    left: 60px !important;
    position: fixed;
}
.bot {
  padding: 5px;
  margin: 10px;
  border-radius: 50%;
  background-color: #ff1100;
  border: 2px solid black;
  width: 50px;
  height: 50px;
  font-size: 25px;
  color: white;
  cursor: pointer;
}
.containerr:hover {
  opacity: 1.0;
  transition: opacity .1s ease-in-out;
  -moz-transition: opacity .1s ease-in-out;
  -webkit-transition: opacity .1s ease-in-out;
}
#helppp{
    visibility: hidden;
    transition: opacity .1s ease-in-out;
  -moz-transition: opacity .1s ease-in-out;
  -webkit-transition: opacity .1s ease-in-out;
}
</style>
<div class="containerr">
<div id="hov" class="vertical--center">
<button id="helpp" onclick="$('#myModal').modal();" class="bot"><i class="fa fa-share"></i></button>
</div>
</div>
<div style="width:100%;hight:100%;">
<div id="game"></div>
      </div>
<script type="text/javascript">
    EJS_player = '#game';
    EJS_biosUrl = '';
    EJS_gameUrl = 'https://tweakdoor.com/game/test/me.gba'; // Url to Game rom
    EJS_core = 'gba';
    EJS_pathtodata = 'data/'; //path to all of the wasm and js files. MUST all be in the same directory!!
    EJS_AdUrl = 'https://tweakdoor.com/ads.html'; //path to AD page
</script>
<script src="data/loader.js"></script>


this is my code please check if eveything if fine?

**Comment by tweakdoor** [comment]
i guess there should be a pop up with x to close it

**Comment by allancoding** [comment]
Yes there is on mine.

**Comment by allancoding** [comment]
You can remove:
````
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script> <script src="js/main.js"></script> <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
<style> .containerr { top: 0; left: 0; position: fixed; z-index: 99999999; opacity: 0; transition: opacity .1s ease-in-out; -moz-transition: opacity .1s ease-in-out; -webkit-transition: opacity .1s ease-in-out; }
.vertical--center {
position: fixed;
top: 0;
left: 0;
width: 150px;
height: 180px;
}
.a{
left: 60px !important;
position: fixed;
}
.bot {
padding: 5px;
margin: 10px;
border-radius: 50%;
background-color: #ff1100;
border: 2px solid black;
width: 50px;
height: 50px;
font-size: 25px;
color: white;
cursor: pointer;
}
.containerr:hover {
opacity: 1.0;
transition: opacity .1s ease-in-out;
-moz-transition: opacity .1s ease-in-out;
-webkit-transition: opacity .1s ease-in-out;
}
#helppp{
visibility: hidden;
transition: opacity .1s ease-in-out;
-moz-transition: opacity .1s ease-in-out;
-webkit-transition: opacity .1s ease-in-out;
}
</style>
<div class="containerr">
<div id="hov" class="vertical--center">
<button id="helpp" onclick="$('#myModal').modal();" class="bot"><i class="fa fa-share"></i></button>
</div>
</div>
```

**Comment by allancoding** [comment]
Sorry Do a Ctrl+Shift+R

**Comment by tweakdoor** [comment]
nothing still

**Comment by allancoding** [comment]
Add this to your page:
````
<style>
body, html{
margin: 0;
padding: 0;
}
</style>
````

**Comment by tweakdoor** [comment]
now its showing loading icon and when the game start its hidden but not showing ads only loading icon

**Comment by tweakdoor** [comment]
you can check https://tweakdoor.com/game/test/me.html

**Comment by allancoding** [comment]
Try clearing your cache

**Comment by allancoding** [comment]
redownload the emulator JS file It is the wrong file redownload it with the one from this page:
[https://github.com/ethanaobrien/emulatorjs/raw/main/data/emulator.js](https://github.com/ethanaobrien/emulatorjs/raw/main/data/emulator.js)

**Comment by tweakdoor** [comment]
hey bro sorry i was busy with family holiday when u will be free?

**Comment by tweakdoor** [comment]
hey bro i need ur help for netplay can u guide me?

**Comment by ethanaobrien** [comment]
Update all emulator files (download data directory) and add the line

```
EJS_AdUrl = ''; //link to ad page
```
